samples/javascript_nodejs/16.proactive-messages/README.md

```diff
@@ -18,6 +18,9 @@ if a bot requires some time to compile a response to the user's question, it may
 and allow the conversation to continue in the meantime. When the bot finishes compiling the response to the
 question, it will share that information with the user.
 
+This project has a NotifyController with an endpoint that will trigger the proactive messages to be sent to 
```

Comment:
This is probably better represented in the node world with "This project has a notify endpoint..." or something other than "notifycontroller".

Reply:
Thanks for pointing that out.